packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -116,10 +137,200 @@ export default class VTable {
                     cell.td.style.borderBottom = getBorderStyle(format.bottomBorderColor);
                     cell.td.style.borderLeft = getBorderStyle(format.verticalBorderColor);
                     cell.td.style.borderRight = getBorderStyle(format.verticalBorderColor);
+                    cell.td.style.borderBottom = getBorderStyle(format.bottomBorderColor);
+                    cell.td.style.backgroundColor = format.bgColor;
                 })
         );
     }
 
+    /**
+     * Set color to even and odd rows
+     * @param format
+     */
+    private setTableRowColor(format: Partial<TableFormat>) {
+        if (!format.bandedRows) {
+            return;
+        }
+        this.cells.forEach((row, index) => {
+            if (index % 2 === 0) {
+                row.forEach(cell =>
+                    cell.td && format.bgColorEven
+                        ? (cell.td.style.backgroundColor = format.bgColorEven)
+                        : ''
+                );
+            } else {
+                row.forEach(cell =>
+                    cell.td && format.bgColorOdd
+                        ? (cell.td.style.backgroundColor = format.bgColorOdd)
+                        : ''
+                );
+            }

```

Comment:
This can be merged into:
const color = index % 2 === 0 ? format.bgColorEven : format.bgColorOdd;
fow.forEach(cell => {
  if (cell.td && color) {
     cell.td.style.backgroundColor = color;
  }
});